src/configuration/utils.ts

```diff
@@ -59,3 +59,12 @@ export const isProductSellableInApp = (
 
   return product.distributionChannel.some((channel) => channel === 'app');
 };
+
+export const removeProductAliasDuplicates = (
```

Comment:
May be able to use `onlyUniquesBasedOnField` with slight modification

Reply:
Could solve it like this, I guess: https://github.com/AtB-AS/mittatb-app/pull/4690